test/hotspot/jtreg/serviceability/jvmti/GetObjectMonitorUsage/ObjectMonitorUsage.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ *
+ * @summary converted from VM Testbase nsk/jvmti/GetObjectMonitorUsage/ObjectMonitorUsage.
+ * VM Testbase keywords: [quick, jpda, jvmti, noras]
+ * VM Testbase readme:
+ * DESCRIPTION
+ *     The test checks if JVMTI function GetObjectMonitorUsage returns
+ *     the expected values for the owner, entry_count, water_count
+ *     fields of JVMTI_monitor_info. The tescases are the following:
+ *       - unowned object without any waitings
+ *       - owned object without any waitings
+ *       - unowned object with waitings through Object.wait()
+ *       - unowned object has been waiting
+ * COMMENTS
+ *     Fixed according to 4669812 bug.
+ *     Ported from JVMDI test nsk/jvmdi/GetMonitorInfo/getmoninfo003.
+ * @requires vm.jvmti
+ * @compile ObjectMonitorUsage.java
+ * @run main/othervm/native -agentlib:ObjectMonitorUsage ObjectMonitorUsage
+ */
+
+public class ObjectMonitorUsage {
+
+    final static int JCK_STATUS_BASE = 95;
+    final static int NUMBER_OF_ENTERING_THREADS = 4;
+    final static int NUMBER_OF_WAITING_THREADS  = 4;
+    final static int NUMBER_OF_THREADS = NUMBER_OF_ENTERING_THREADS + NUMBER_OF_WAITING_THREADS;
+
+    static Object lockCheck = new Object();
+    static Thread thr[] = new Thread[NUMBER_OF_THREADS];
```

Comment:
Nit: the `[]` go on the type not the variable.

Reply:
Thanks, fixed now.